plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcColumnHandle.java

```diff
@@ -162,13 +172,14 @@ public static Builder builderFrom(JdbcColumnHandle handle)
         return new Builder(handle);
     }
 
-    public static class Builder
+    public static final class Builder
```

Comment:
The 1st commit added removed final, and the 2nd commit restored it. Please completely remove the change.